src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -358,7 +358,7 @@ void Renderer::Impl::render(const UpdateParameters& updateParameters) {
                 // We only need clipping when we're _not_ drawing a symbol layer. The only exception
                 // for symbol layers is when we're rendering still images. See render_symbol_layer.cpp
                 // for the exception we make there.
-                if (!symbolLayer || parameters.mapMode == MapMode::Still) {
+                if (!symbolLayer || parameters.mapMode != MapMode::Tile) {
```

Comment:
We don't clip symbol layers at all anymore, right? So should this just be `if (!symbolLayer)`, and the comment should be updated to remove the reference to the still image exception?